docs/en/enterprise-edition/content-collections/governance/custom-build-policies/custom-build-policy-examples.adoc

```diff
@@ -701,3 +701,19 @@ definition:
     - "[A-Za-z0-9]{8,20}"
     - "my-super-secret-password-regex"
 ----
+
+==== Example #2 Detect Passwords
+
+This policy detects instances where literal strings starting with "password" or "pwd" are used directly in the code or configuration files. This is a security risk because it exposes the actual password value.
```

Comment:
```suggestion
This policy detects instances where literal strings starting with "password", "pass" or "pwd" are used directly in the code or configuration files. This is a security risk because it exposes the actual password value.
```